### PR TITLE
Improve style

### PR DESCRIPTION
**Pull request by Ticki**

### CODE REVIEW

**Comment by hauleth** [comment]
Do not edit code here. Use [website](redox-os/website). This is read-only repo.

File: blog/hello-world/index.html [hunk 1]
```diff
@@ -32,7 +32,7 @@
       </a>
     </div>
     <div class="content">
-      <h2 class="post-title"><a href="http://redox-os.github.io/blog/hello-world/">And So It Begins…</a></h2>
+      <h2 class="post-title"><a href="#">And So It Begins…</a></h2>
```
Comment (hauleth) [inline blog/hello-world/index.html]: Why so?

**Comment by hauleth** [comment]
We use [Hugo](http://gohugo.io/) as a blog engine so you can read docs about how it works.

**Comment by Ticki** [comment]
Oh sorry. I didn't notice. My bad.

**Comment by Ticki** [comment]
Will send a PR for the `website` repo.

**Comment by hauleth** [comment]
No problem. You will get used to it :smile:

**Pull request closed without merging**